profile.go

```diff
@@ -126,6 +126,25 @@ Make sure to backup your data frequently.`,
 			return nil
 		},
 	},
+	"postgresds": {
+		Description: `Replaces default datastore configuration with experimental
+postgres datastore.
+
+WARNING: postgres datastore is experimental.
+Make sure to backup your data frequently.`,
+		Transform: func(c *Config) error {
+			c.Datastore.Spec = map[string]interface{}{
+				"type":   "measure",
+				"prefix": "postgres.datastore",
+				"child": map[string]interface{}{
+					"type":       "postgres",
+					"path":       "postgres",
+					"syncWrites": true,
```

Comment:
Does the "syncWrites" option mean anything to postgres?

Reply:
Looks like it's not handled in https://github.com/ipfs/go-ipfs/pull/5403/files either, so this can be just dropped

Reply:
Dropped the syncWrites an updated go-ipfs-addr and go-libp2p-peer dependencies. Can this be merged?